docs/instaslice-emulator-openshift-4.18.md

```diff
@@ -0,0 +1,141 @@
+
+#### Add OpenShift 4.17 Catalog
+
+1. **Create a namespace for the Cert Manager Operator**:
+
+   ```bash
+   oc create namespace cert-manager-operator
+   ```
+
+2. **Add the 4.17 catalog source**:
+
+   ```yaml
+   apiVersion: operators.coreos.com/v1alpha1
+   kind: CatalogSource
+   metadata:
+     name: rh-operators-417
+     namespace: openshift-marketplace
+   spec:
+     sourceType: grpc
+     image: registry.redhat.io/redhat/redhat-operator-index:v4.17
+     displayName: Red Hat 4.17 Catalog
+     publisher: Red Hat
+     updateStrategy:
+       registryPoll:
+         interval: 1m
+   ```
+
+   Apply this YAML file:
+
+   ```bash
+   oc apply -f <path_to_yaml_file>
+   ```
+
+3. **Create a subscription for the Cert Manager Operator**:
+
+   ```yaml
+   apiVersion: operators.coreos.com/v1
+   kind: OperatorGroup
+   metadata:
+     name: openshift-cert-manager-operator
+     namespace: cert-manager-operator
+   spec:
+     targetNamespaces:
+     - "cert-manager-operator"
+   ---
+   apiVersion: operators.coreos.com/v1alpha1
+   kind: Subscription
+   metadata:
+     name: openshift-cert-manager-operator
+     namespace: cert-manager-operator
+   spec:
+     channel: stable-v1
+     name: openshift-cert-manager-operator
+     source: rh-operators-417
+     sourceNamespace: openshift-marketplace
+     installPlanApproval: Automatic
+     startingCSV: cert-manager-operator.v1.14.0
+   ```
+
+   Apply this YAML file:
+
+   ```bash
+   oc apply -f <path_to_yaml_file>
+   ```
+
+4. **Verify that the Cert Manager Operator is working**:
+
+   ```bash
+   $ oc get pods -n cert-manager-operator
+   NAME                                                        READY   STATUS    RESTARTS   AGE
+   cert-manager-operator-controller-manager-77746bc67c-skl7k   2/2     Running   0          63m
+   ```
+
+   You should see similar outputs:
+
+   ```bash
+   $ oc get pods -n cert-manager
+   NAME                                       READY   STATUS    RESTARTS   AGE
+   cert-manager-5d59ff6b86-67s2l              1/1     Running   0          62m
+   cert-manager-cainjector-6c54b88c7d-dkkbr   1/1     Running   0          63m
+   ```
+
+### 2. Deploy InstaSlice in Emulator Mode
+
+To deploy the InstaSlice operator in emulated mode on OpenShift:
+
+1. **Deploy the operator in emulator mode**:
+
+   Use the following command to deploy the operator:
+
+   ```bash
+   make ocp-deploy-emulated
+   ```
+
+2. **Add GPU capacity to the cluster**:
+
+   Apply the fake GPU capacity configuration:
+
+   ```bash
+   kubectl apply -f test/e2e/resources/instaslice-fake-capacity.yaml
+   ```
+
+3. **Verify that the InstaSlice object exists**:
+
+   ```bash
+   kubectl describe instaslice
+   ```
+
+4. **Submit an emulator pod**:
+
+   Use the following command to submit an emulator pod:
+
+   ```bash
+   kubectl apply -f samples/emulator-pod.yaml
```

Comment:
nit: since we are working against openshift, steps 2,3,4 should probably use `oc`.

Reply:
updated, thanks.